ui/tutorial-renderer/src/main/java/org/cru/godtools/tutorial/layout/TipsLayout.kt

```diff
@@ -26,112 +28,137 @@ import com.airbnb.lottie.compose.LottieCompositionSpec
 
-        Text(
-            stringResource(body),
-            style = MaterialTheme.typography.bodyMedium,
-            textAlign = TextAlign.Center,
-            modifier = Modifier.padding(top = 16.dp)
-        )
+            )
+            LottieAnimation(
+                composition,
+                { progress },
+                modifier = Modifier
+                    .height(290.dp)
+                    .fillMaxWidth()
+            )
 
-        if (body2 != null) {
+            Text(
+                text = stringResource(title),
+                style = MaterialTheme.typography.titleLarge,
+                textAlign = TextAlign.Center,
+                modifier = Modifier.padding(horizontal = 32.dp)
+            )
 
             Text(
-                // Spacer(modifier = Modifier.minLinesHeight(minLines = 1, textStyle = MaterialTheme.typography.bodyMedium)),
-                stringResource(body2),
+                stringResource(body),
                 style = MaterialTheme.typography.bodyMedium,
-                textAlign = TextAlign.Center
+                textAlign = TextAlign.Center,
+                modifier = Modifier.padding(horizontal = 32.dp)
             )
-        }
-        if (anim == R.raw.anim_tutorial_tips_light) {
-            Spacer(Modifier.padding(top = 150.dp))
-            Button(
-                onClick = {
-                    onTutorialAction(R.id.action_tips_finish)
-                },
 
-                modifier = Modifier.width(250.dp)
-            ) {
-                Text(
-                    text = stringResource(id = R.string.tutorial_tips_action_start),
-                    fontSize = 30.sp
+            if (body2 != null) {
+                Spacer(
+                    modifier = Modifier.minLinesHeight(
+                        minLines = 1,
+                        textStyle = MaterialTheme.typography.bodyMedium
+                    )
                 )
-            }
-        } else {
-            Button(
-                onClick = nextPage,
-                modifier = Modifier
-                    .padding(top = 150.dp)
-                    .width(250.dp)
-            ) {
                 Text(
-                    text = stringResource(id = R.string.tutorial_tips_action_continue),
-                    fontSize = 30.sp
+                    stringResource(body2),
+                    style = MaterialTheme.typography.bodyMedium,
+                    textAlign = TextAlign.Center,
+                    modifier = Modifier.padding(horizontal = 32.dp)
                 )
             }
         }
+        Box(
+            modifier = Modifier.weight(1f)
+        ) {
+            if (page == Page.TIPS_START) {
+                Spacer(Modifier.padding(top = 150.dp))
+                Button(
+                    onClick = {
+                        onTutorialAction(R.id.action_tips_finish)
+                    },
+
+                    modifier = Modifier.padding(horizontal = 32.dp).width(275.dp)
```

Comment:
there is a `Modifier.align(Alignment.BottomCenter)` that will put the button on the bottom of the box